src/server/service/debug.rs

```diff
@@ -546,8 +569,137 @@ where
         self.debugger.reset_to_version(req.get_ts());
         sink.success(ResetToVersionResponse::default());
     }
+
+    fn flashback_to_version(
+        &mut self,
+        ctx: RpcContext<'_>,
+        req: FlashbackToVersionRequest,
+        sink: UnarySink<FlashbackToVersionResponse>,
+    ) {
+        let tmp = self.region_info_accessor.region_leaders();
+        let region_leaders = tmp.read().unwrap();
+
+        // Prepare flashback for specified regions.
+        let start_ts = self
+            .pool
+            .block_on(self.pd_client.get_tso())
```

Comment:
As the entire task is within an async task pool, using `block_on` here should be avoided as it could potentially block the entire runtime.

Reply:
fixed